docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,90 @@
 ---
-title: "Administration Console"
+title: "Use the Remote Console"
 date: 2019-02-23T17:39:15-05:00
-draft: true
+draft: false
 weight: 2
+description: "Use the Oracle WebLogic Server Remote Console to manage a domain running in Kubernetes."
 ---
+
+The Oracle WebLogic Server Remote Console is a lightweight, open source console that does not need to be collocated with a WebLogic Server domain.
+You can install and run the Remote Console anywhere. For an introduction, read the blog, ["The NEW WebLogic Server Remote Console"](https://blogs.oracle.com/weblogicserver/new-weblogic-server-remote-console).
+
+For detailed documentation, see the [Oracle WebLogic Server Remote Console](https://github.com/oracle/weblogic-remote-console) GitHub project.
+
+A major benefit of using the Remote Console is that you don't need to install or run the WebLogic Server Administration Console on WebLogic Server instances.
+You can use the Remote Console with WebLogic Server **slim** installers, available on the [OTN](https://www.oracle.com/middleware/technologies/weblogic-server-installers-downloads.html)
+or [OSDC](https://edelivery.oracle.com/osdc/faces/Home.jspx;jsessionid=LchBX6sgzwv5MwSaamMxrIIk-etWJLb0IyCet9mcnqAYnINXvWzi!-1201085350).
+Slim installers reduce the size of WebLogic Server downloads, installations, container images, and Kubernetes pods.
+For example, a WebLogic Server 12.2.1.4 slim installer download is approximately 180 MB smaller.
+
+## Use the Remote Console
+
+To access WebLogic Server domains running in Kubernetes:
+
+1. Install, configure, and start the Remote Console according to these [instructions](https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md).
+
+1. For the Remote Console to access the Administration Server running in Kubernetes, you can:
+
+   * Use [curl](#use-curl).
```

Comment:
This curl command is to connect to the on-server  admin console. Initially it is for internal testing to make sure we have the right LBPORT for path routing rule before we use the same LBPORT on remote console . For the public documentation, we don't need this curl command.

Reply:
OH.

Reply:
Yesterday I had zoom session with Tom Moreau to confirm my understanding about curl.

Reply:
Also I saw in the new path routing rule file we have ONLY rule for remote console, which is good. That means this curl command to connect to the on-premise admin console will not work.